modules/integration/src/test/scala/scala/cli/integration/TestTestDefinitions.scala

```diff
@@ -509,5 +509,4 @@ abstract class TestTestDefinitions(val scalaVersionOpt: Option[String])
         expect(output.contains("Hello from native"))
     }
   }
-
```

Comment:
It'd be nice to check that tests run fine with `--jvm 8` too (this should ensure that `test-runner` is fine on Java 8).

Reply:
Done